content/blog/2021/07/camel-jbang/index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Camel + JBang: bringing the world of Java scripting to Camel"
+date: 2021-07-26
+authors: ["orpiske"]
+categories: ["Features", "Camel"]
+preview: "Camel + JBang"
+summary: "Bringing the world of Java scripting to Camel"
+---
+
+One of the new features we are introducing with Camel 3.12 is a command-line utility that leverages [JBang](https://www.jbang.dev/) in order to make it possible to benefit from the large set of Apache Camel features, without the need of writing a full blown application.
+
+# What is JBang?
+
+Before showing what it can do, it's important to give a brief overview of what JBang is. According to their website, JBang lets us " … create, edit and run self-contained source-only Java programs with unprecedented ease". In practice, this means that it lets us run Java applications without the need to write an application, going through the usual build/compile/deploy cycle. It turns Java into a scripting language, in a similar fashion to Python, Shell Script and others. With easy access to the dependency management, configuration, debugging and other features, it makes it very easy for Java developers to use all the power that Java and its ecosystem of components can offer.
+
+# Camel + JBang
+
+We are leveraging the set of features provided by JBang to expose and simplify access for some of the Apache Camel features with the intention to simplify trying Camel; make it easier to learn about and search for components, languages and features; and provide mechanisms for the users to try agile integration.
+
+To showcase the features that have been already implemented, we prepared a set of screencasts showing the installation process, the
+search functionality and running a few routes.
+
+*Note*: At the moment we are writing this post, Camel 3.12 has not been released yet. As such, by installing the CLI utility, you are effectively getting access to a development version of Camel. Although we are excited to share news about new features, we also ask you to be please use caution and do not run this on production environments and also keep in mind that as a preview, features may change.
+
+# Installation
+
+As with any tool, it's nice if it's easy to install. JBang makes it easy for us by providing an installation feature that works well with git, Github. If you have JBang installed on your system, then to install CamelJBang is as easy as running:
+
+```
+jbang app install CamelJBang@apache/camel
+```
+
+The screencast below shows it in practice:
+
+[![asciicast](https://asciinema.org/a/426328.svg)](https://asciinema.org/a/426328)
```

Comment:
I think we'll need to add asciicinema.org to the CSP, here:

https://github.com/apache/camel-website/blob/8aea817dc90940c80b58e066f9df71d02b59877a/static/.htaccess#L1568

Good way to test is to run [Apache HTTPD locally](https://github.com/apache/camel-website/tree/main/support).

Reply:
Thanks! 

I tried doing adding `https://asciinema.org` to the img-src part but it didn't work here. Instead of risking breaking the site, I just added the images to the post and retained the link.